example-dotnet/Program.cs

```diff
@@ -18,43 +19,45 @@ static void Main(string[] args)
             Say.hello("example");
             Console.WriteLine("Hello World!");
 
-	    // Usage of BCrypt.Net API
-
             // The following API call is meant to demonstrate
             // vulnerable methods (the simplest case).
               
             var hashedPassword = BCrypt.Net.BCrypt.HashPassword("my-secret-password");
             BCrypt.Net.BCrypt.Verify("wrong-password", hashedPassword);
 
-            // The next API calls to four libraries are meant to
-            // demonstrate update advisor and auto pull request
-            // features.
+            // The following API calls are meant to demonstrate update
+            // advisor and auto pull request features. The following
+            // is the detail of the libraries and the expected
+            // results.
             //
-            // The first two are the cases where the update versions
+            // The following is the cases where the update versions

```

Comment:
Grammar. Really minor.
```suggestion
            // The following are the cases where the update versions
```